grid2op/Environment/baseEnv.py

```diff
@@ -334,6 +334,7 @@ def __init__(
         highres_sim_counter=None,
         update_obs_after_reward=False,
         n_busbar=2,
+        allow_shedding:bool=False,
```

Comment:
`allow_shedding= False` should rather be `allow_shedding = DEFAULT_ALLOW_SHEDDING` (plus the first comment I made about the name "shedding"